### ISSUE TITLE
Differences in peak gain

### ISSUE THREAD

**Issue by girtabb** [ShanonPearce/ASH-Listening-Set#8]
Hi again, I'm really loving this project and how far it has come. However, there's on thing I would like to bring up. Each HRIR has it's own peak volume and so in order to eliminate clipping, you have to change the preamp according to the room every time you switch them. I know, there's tools like peace UI that solve that problem but I was wondering if you could make the room files have a more equal peak gain. Thanks in advance

**Comment by ShanonPearce** [comment]
Ill see what I can do. The rooms were level matched in the low frequencies however the peak gain is expected to vary a little depending on the shape of the low frequency spectrum. The best approach for now might be to apply a sufficient level of negative gain to eliminate clipping in all rooms.

**Comment by girtabb** [comment]
Yeah, the only thing is I have issues with getting really high volumes out of my amplifier. The fluctuations in the FR are spiky as a result of reflections but it's totally worth it. I'll trade sound quality over volume headroom any day but I just think leveling them to a more equal peak loudness shouldn't be an issue
